apps/base-docs/docs/account-abstraction-with-base-paymaster/intro-to-account-abstraction.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Intro to Account Abstraction
+description: Learn what Account Abstraction it, how it works and what problems it solves.
+hide_table_of_contents: false
+---
+
+ERC-4337, also known as "Account Abstraction", is a standard that allows smart contracts to initiate transactions, thus enabling any logic that users want to implement to be encoded into the smart contract wallet itself for execution on Ethereum.
+
+Account Abstraction has the potential to be a massive game changer for onchain user experience and many believe it will play a key role in bringing the next billion users onchain.
+
+---
+
+## Prerequisites
+
+Before reading this guide, you should:
+
+- Be familiar with modern web development
+- Possess a general understanding of the EVM and smart contracts
+
+---
+
+## Objectives
+
+By the end of this lesson you should be able to:
+
+- Explain what Account Abstraction is and what problems it solves, primarily onchain UX.
+- Describe the difference between contract accounts and EOAs and the limitations of contract accounts (i.e. they cannot initiate transactions) and how 4337 solves that particuilarly with `UserOperation`s.
+- Outline how Account Abstraction works, particularly how users interact with smart contract wallets.
+
+---
+
+## The Problem Account Abstraction Solves
+
+Onchain applications are difficult to use for most as they require a lengthly onboarding process for the average internet user who isn't used to being onchain.
+
+For example, here are the prelimenary steps a user needs to go through before they can use an onchain app:
+
+1. Create an account/learn to use a wallet (Coinbase Wallet, Metamask, etc).
+2. Store private key.
+3. Initiate a transaction in the onchain app.
+4. Realize that transactions cost gas.
+5. Learn what gas it.
```

Comment:
what gas is?

Reply:
I'll expand - most new users don't know what gas is.